pysat/_files.py

```diff
@@ -480,7 +480,7 @@ def from_os(cls, data_path=None, format_str=None,
         format_str : string with python format codes
             Provides the naming pattern of the instrument files and the
             locations of date information so an ordered list may be produced.
-            Supports 'year', 'month', 'day', 'hour', 'min', 'sec', 'version',
+            Supports 'year', 'month', 'day', 'hour', 'mins', 'sec', 'version',
```

Comment:
This is very much responsive to my comment about min being a less than great variable choice. I'm not sure I feel the same way about its use as a format keyword. From a UI perspective 'mins' is now an odd keyword that breaks the naming pattern.

Reply:
updated to minute and second